newsletter-template.md

```diff
@@ -0,0 +1,35 @@
+> Dat Project Newsletter Template
+
+![Dat Project logo](https://datproject.github.io/design/downloads/dat-data-logo.svg)
+# Dat Project Newsletter
+_April 2019_
+
+## Three Protocols and a Future of the Decentralized Internet
+_by Darius Kazemi_
+
+"If what we broadly want is for power to be redistributed from the hands of large centralized corporate interests and back into the hands of individual people and small groups, then we already have the technical protocols to do this."
+
+<a class="btn btn--full btn--green" href="https://blog.datproject.org/2019/03/22/three-protocols-and-a-future-of-the-decentralized-internet/">READ MORE</a>
+
+## Dat Comm-Comm
+
+comm-comm meetings are a community effort to create a space to talk about Dat, share updates, make announcements, & meet people working with the Dat community. Huge thanks to organizers **Martin Heidegger** & **Diego Paez** for starting comm-comm!
+
+<a class="btn btn--full btn--green" href="https://blog.datproject.org/2019/03/21/comm-comm-what/">LEARN MORE ABOUT COMM-COMM</a>
+
+## Dat Field Notes
+
+_Dat Field Notes_ is an informal collection of resources, notes describing Dat, how to use Dat, and how the Dat Project is run.
+
+<a class="btn btn--full btn--green" href="https://blog.datproject.org/2019/03/19/dat-field-notes/">READ MORE!</a>
+
+
+### [__Now Hiring a Developer__](https://blog.codeforscience.org/css-jobs-mlab-openreview/) to revolutionize peer review!
```

Comment:
Not sure if this should be on the dat list since it's a CSS thing. We could consider cross posting in general, which is fine, just want to flag that it should be done sparingly to not further confuse dat and css

Reply:
Makes sense. I just copied @joehand's first newsletter here for an example: https://mailchi.mp/881088ed65b0/dat-news-announcements-three-protocols-and-a-future-of-the-decentralized-internet-comm-comm-and-more

I'll snip that bit out for now 👍🏻